### DISCUSSION TITLE
Fullscreen option not available for classic projects

### DISCUSSION THREAD

**Discussion by pandafy** [community/community#58445] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

The "fullscreen" option for the classic projects does not appear when the user is logged in. But, it does appear (and work) when viewing a public project board. 

Classic project when the user is logged in 

![Screenshot from 2023-06-19 15-19-25](https://github.com/community/community/assets/32094433/061f71b2-7ace-4c94-b7fd-f9a032729c97)

Classic project when the user is **NOT** logged in 

![image](https://github.com/community/community/assets/32094433/fc2e30ce-8348-42fd-ac49-923cca2a8810)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by RalfJung** [comment]
This is a recent regression, right?  I have used fullscreen view for my private project for years, but now the button is gone (I even still have the `?fullscreen=true` in my URL in the tab I have always open, but it doesn't do anything any more). I hope this feature is brought back; right now about 1/3 of the available space is wasted, making it much harder to get an overview over the project.

**Reply by pandafy** [reply]
Yes, this is a regression. Right now, I am using [User JavaScript and CSS browser extension](https://chrome.google.com/webstore/detail/user-javascript-and-css/nbhcbdghjpllgmfilhnhkllmkecfmpld/related) to hide the app header on the classic projects. I have already shared my feedback for the [navigation ui](https://github.blog/2023-06-15-exploring-github-with-the-redesigned-navigation-now-in-public-beta/). 

Alternatively, you can disable the new navigation UI from the [feature preview settings](https://docs.github.com/en/get-started/using-github/exploring-early-access-releases-with-feature-preview#exploring-beta-releases-with-feature-preview).
